config/admin-cli.properties

```diff
@@ -19,11 +19,43 @@ cli.controller.grpc.uri=tcp://localhost:9090
 cli.store.metadata.backend=segmentstore
 
 # General configuration for the Pravega cluster at hand.
-pravegaservice.cluster.name=pravega/pravega
+pravegaservice.clusterName=pravega/bar-pravega
```

Comment:
Why did you change the property name?

Reply:
This was wrong. Name should be `clusterName` as [used here]([url](https://github.com/pravega/pravega/blob/34ddf4935faf6edc4cd1d2c38cbbec77486e6dc4/segmentstore/server/src/main/java/io/pravega/segmentstore/server/store/ServiceConfig.java#L54)).